instrumentation/opentelemetry-instrumentation-redis/src/opentelemetry/instrumentation/redis/__init__.py

```diff
@@ -128,6 +128,10 @@ def response_hook(span, instance, response):
 
 
 def _set_connection_attributes(span, conn):
```

Comment:
Nit: Can probably rename `conn` to instance since we could be dealing with connections or clusters now.